lib/krb5/fcache.c

```diff
@@ -528,7 +528,7 @@ fcc_initialize(krb5_context context,
 {
     krb5_fcache *f = FCACHE(id);
     int ret = 0;
-    int fd;
+    int fd = -1;
```

Comment:
fd is set to -1 upon entry to fcc_open() at line 538.  As there is no use of 'fd' before then this is unnecessary.